### PR TITLE
Use declared TTL for stat cache instead of default value for normal cache

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/pipecd/server.go [hunk 1]
```diff
@@ -65,6 +65,7 @@ var (
 
 const (
 	defaultPipedStatHashKey = "HASHKEY:PIPED:STATS"
+	defaultPipedStatTTL     = 2 * time.Minute
```
Comment (nakabonne) [inline cmd/pipecd/server.go:68]: This shouldn't be changeable even in the future; `pipedStatTTL` may be better to prevent us confusion.

**Comment by nakabonne** [comment]
This change avoids showing wrong metrics for a long time. LGTM except for a tiny comment!
/lgtm

File: cmd/pipecd/server.go (hunk 1 above)
Comment (khanhtc1202) [reply cmd/pipecd/server.go:68]: Nice point 👍 lets me adopt that 🙆‍♂️

Comment (khanhtc1202) [reply cmd/pipecd/server.go]: Done

**Review by nghialv — APPROVED** [review]

**Pull request merged**